06_source/ADAS/ADAS_MCU/ADAS_Cfg.h

```diff
@@ -39,45 +53,89 @@
 #define SICK_NAK                                        0x15
 
 //Pins
+/** PWM for Left Motor*/
 #define PIN_MTR_L_PWM                                   11
+/** PWM for Right Motor*/
 #define PIN_MTR_R_PWM                                   12
+/** Enable for Right and Left Motor*/
 #define PIN_ENABLE                                      52
+/** Direction for Left Motor*/
 #define PIN_DIRECTION_L                                 50
+/** Direction for Right Motor*/
 #define PIN_DIRECTION_R                                 48
+/** Encoder Interrupt for Right Motor*/
 #define PIN_ENC_R                                        2
+/** Encoder Interrupt for Left Motor*/
 #define PIN_ENC_L                                        3
+/** External Start Button*/
 #define Btn_Start                                       46
+/** Voltage Level of Battery*/
 #define PIN_VBAT                                        A0
 
 
 //Navigation parameters
```

Comment:
Navigation parameters not needed on the MCU

Reply:
removed